react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -0,0 +1,41 @@
+# Next.js. Server Side Rendering
+
+## Technical requirements
+
+1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
+2. Next.js Pages API Integration:
+   - Migrate your application from vite to next.js by using Pages folder API. You'll need to adapt existing libraries to work with next.js.
```

Comment:
Not sure which libraries need to be updated. May be it is worth adding `if necessary` to avoid additionla questions? :)